lab_1_classify_by_unigrams/main.py

```diff
@@ -80,6 +149,18 @@ def preprocess_profile(profile: dict) -> dict[str, str | dict] | None:
     :return: a dict with a lower-cased loaded profile
     with relative frequencies without unnecessary ngrams
     """
+    if not (isinstance(profile, dict)
+            and 'name' in profile
+            and 'freq' in profile
+            and 'n_words' in profile):
+        return None
+    freq_dict = {}
+    for unigram in profile['freq']:
+        if unigram.lower() in freq_dict:
```

Comment:
`unigram.lower()`,  `profile['n_words'][0]`, `profile['freq'][unigram]` to variables, please

Reply:
fixed